libs/community/langchain_community/vectorstores/opensearch_vector_search.py

```diff
@@ -401,6 +402,73 @@ def _default_painless_scripting_query(
     }
 
 
+def _default_hybrid_search(
+    query_text: str, query_vector: List[float], k: int = 4
+) -> Dict:
+    """Returns payload for performing hybrid search for given options.
+    Args: options: dict containing the following
```

Comment:
nit:https://google.github.io/styleguide/pyguide.html#244-decision  

take a look at the URL above that shows google style doc-strings

the spec is whitespace sensitive as far as I know, so it should be a new line after Args:, a new line before Args:, and a new line after Returns: